tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -564,11 +481,13 @@ function configure_containerd() {
 function remove_artifacts() {
 	echo "deleting kata artifacts"
 	rm -rf /opt/kata/*
+
+	if [[ "${CREATE_RUNTIMECLASSES}" == "true" ]]; then
```

Comment:
Wouldn't be better to move this block to `cleanup_cri_runtime()` to keep the same flow? I don't really know the consequences of removing the runtimeclasses only at remove_artifacts when everything else is supposed to be reestablished.

Is it what you mean on your comment @ldoktor ?

Reply:
The runtime class is a kubernetes type, you can create a runtime class at any time, and remove it at any time.
So, really doesn't make any difference where it is.

Reply:
```bash
function delete_runtimeclasses() {
        echo "Deleting the runtime classes" 

        for shim in "${shims[@]}"; do
                echo "Deleting the kata-${shim} runtime class"
                kubectl delete -f /opt/kata-artifacts/runtimeclasses/kata-${shim}.yaml
        done


        if [[ "${CREATE_DEFAULT_RUNTIMECLASS}" == "true" ]]; then
                echo "Deleting the kata runtime class for the default shim (an alias for kata-${default_shim})"
                cp /opt/kata-artifacts/runtimeclasses/kata-${default_shim}.yaml /tmp/kata.yaml
                sed -i -e 's/name: kata-'${default_shim}'/name: kata/g' /tmp/kata.yaml
                kubectl delete -f /tmp/kata.yaml
                rm -f /tmp/kata.yaml
        fi
}
```

As you can see, nothing from the node itself is used.